docs/maui/includes/behavior-bindings.md

```diff
@@ -0,0 +1,2 @@
+> [!IMPORTANT]
+> The .NET MAUI Community Toolkit Behaviors do not set the `BindingContext` of a behavior, because behaviors can be shared and applied to multiple controls through styles. For more information refer to [.NET MAUI Behaviors](/dotnet/maui/fundamentals/behaviors?view=net-maui-8.0#net-maui-behaviors)
```

Comment:
```suggestion
> The .NET MAUI Community Toolkit Behaviors do not set the `BindingContext` of a behavior, because behaviors can be shared and applied to multiple controls through styles. For more information refer to [.NET MAUI Behaviors](/dotnet/maui/fundamentals/behaviors#net-maui-behaviors)
```

Lets remove the net8 specific thing